traffic_ops/app/lib/API/DeliveryServiceServer.pm

```diff
@@ -27,6 +28,11 @@ sub index {
 	my @data;
 	my $orderby = $self->param('orderby') || "deliveryservice";
 
+	#FOR THE REVIEWER - Currently I do not check DS tenancy here.
```

Comment:
With the new roles/capabilities model there will be a lot of flexibility regarding roles and capabilities. For example, we could create a role called ds-admin with the following capabilities: 

- ds-read
- ds-write
- ds-server-read
- ds-server-write
- ds-regex-read
- etc

so that means anyone with this ds-admin role can manage all aspects of a ds, including changing the server assignments.

other companies may only want to attach ds-server-write to the a higher level role but we can't control that so my suggestion is we always check tenancy and filter results accordingly.